integration-tests/gradle/src/test/resources/list-extensions-with-managed-dependency-without-scope/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
```

Comment:
This whole test module seems to be unused. Why does it exist?

Reply:
The maven IT has two tests in it, I was working on the second one but got some errors, so I deleted it from the PR but kept these files

Reply:
Please delete these files as well

Reply:
Okk